engine.py

```diff
@@ -7,12 +7,15 @@
 """
 import os
 import sys
-import ctypes
-import shutil
-import logging
 import traceback
 
+from PyQt4 import QtGui
+
+import sgtk
```

Comment:
Actually you were right about this one. I had used it at first, and then realized than one of the args when calling __init__ is a Tank instance. So no need for the import, I removed it.